appyters/Enrichr_Scatterplot_Appyter/Scatter-Appyter.ipynb

```diff
@@ -83,9 +86,49 @@
     "    'TF_Perturbations_Followed_by_Expression',\n",
     "    'Transcription_Factor_PPIs',\n",
     "    'TRANSFAC_and_JASPAR_PWMs',\n",
-    "    'TRRUST_Transcription_Factors_2019']) }}'\n",
+    "    'TRRUST_Transcription_Factors_2019',\n",
+    "    'ARCHS4_Kinases_Coexp',\n",
+    "    'BioCarta_2016',\n",
+    "    'BioPlanet_2019',\n",
+    "    'BioPlex_2017',\n",
+    "    'CORUM',\n",
+    "    'Elsevier_Pathway_Collection',\n",
+    "    'HMS_LINCS_KinomeScan',\n",
+    "    'HumanCyc_2016',\n",
+    "    'huMAP',\n",
+    "    'KEA_2015',\n",
+    "    'KEGG_2019_Human',\n",
+    "    'KEGG_2019_Mouse',\n",
+    "    'Kinase_Perturbations_from_GEO_down',\n",
+    "    'Kinase_Perturbations_from_GEO_up',\n",
+    "    'L1000_Kinase_and_GPCR_Perturbations_down',\n",
+    "    'L1000_Kinase_and_GPCR_Perturbations_up',\n",
+    "    'NCI-Nature_2016',\n",
+    "    'NURSA_Human_Endogenous_Complexome',\n",
+    "    'Panther_2016',\n",
+    "    'Phosphatase_Substrates_from_DEPOD',\n",
+    "    'PPI_Hub_Proteins',\n",
+    "    'Reactome_2016',\n",
+    "    'SILAC_Phosphoproteomics',\n",
+    "    'SubCell_BarCode',\n",
+    "    'WikiPathways_2019_Human',\n",
+    "    'WikiPathways_2019_Mouse',\n",
+    "    'GO_Cellular_Component_2018',\n",
+    "    'GO_Molecular_Function_2018',\n",
+    "    'Human_Phenotype_Ontology',\n",
+    "    'Jensen_COMPARTMENTS',\n",
+    "    'Jensen_DISEASES',\n",
+    "    'Jensen_TISSUES',\n",
+    "    'Achilles_fitness_decrease',\n",
+    "    'Achilles_fitness_increase',\n",
+    "    'ARCHS4_IDG_Coexp',\n",
+    "    'ClinVar_2019',\n",
+    "    'dbGaP',\n",
+    "    'DepMap_WG_CRISPR_Screens_Broad_CellLines_2019',\n",
+    "    'DepMap_WG_CRISPR_Screens_Sanger_CellLines_2019',\n",
+    "    'DrugMatrix']) }}'\n",
     "\n",
-    "significance_value = {{ StringField(name='significance_value', label='Significance Value', default='', description='Enter a value at which p-values are significant.', section = 'section3') }}"
+    "significance_value = {{ StringField(name='significance_value', label='Significance Value', default='0.05', description='Enter a value at which p-values are significant.', section = 'section3') }}"
```

Comment:
Even though I didn't notice this before, this should really be a FloatField instead of a string field for security reasons

Reply:
Because you use `float(val)` on it later anyway, it's fine for now--but please consider using a floatfield instead for simplicity